orchestra/todos/views.py

```diff
@@ -48,15 +47,17 @@ def update_todos_from_todolist_template(request):
 def worker_recent_todo_qas(request):
     """
     The function returns TodoQAs from the requesting user's most recent
-    task assignment provided they were created in the last 90 days.
+    task assignment with a todo qa.
     """
     try:
-        todoQAs = TodoQA.objects.filter(
-            todo__task__assignments__worker__user=request.user,
-            approved=False,
-            created_at__gte=datetime.now() - timedelta(days=90))
-        todos_recommendation = {todoQA.todo.description: TodoQASerializer(
-            todoQA).data for todoQA in todoQAs}
+        most_recent_worker_task = TodoQA.objects.filter(
+            todo__task__assignments__worker__user=request.user
```

Comment:
may well put `approved=False` here

Reply:
That may be wrong in a scenario where the worker has done everything correctly (all approved). It would seem like we are picking on mistakes. :)